src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/DefaultGATKVariantAnnotationArgumentCollection.java

```diff
@@ -62,6 +40,19 @@ public DefaultGATKVariantAnnotationArgumentCollection(final List<String> default
     @Argument(fullName = StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_ANNOTATIONS, shortName = StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_ANNOTATIONS, doc = "Disable all tool default annotations", optional = true)
```

Comment:
Is this branch properly rebased onto master? Why is it missing the merged patch from here: https://github.com/broadinstitute/gatk/pull/4671/files